### PR TITLE
Improve acceptance test coverage

### PR DESCRIPTION
**Pull request by treydock**
# Pull Request Checklist

## Description

Improve acceptance test coverage

## Related Issue

Relates to #901 

## Motivation and Context

Improve coverage on properties that are either complex or not direct mappings to sensuctl resource properties.

## How Has This Been Tested?

Rake beaker acceptance tests.

### CODE REVIEW

File: spec/acceptance/sensu_asset_spec.rb [hunk 1]
```diff
@@ -7,9 +7,12 @@
       pp = <<-EOS
       include ::sensu::backend
       sensu_asset { 'test':
-        url     => 'http://example.com/asset/example.tar',
-        sha512  => '4f926bf4328fbad2b9cac873d117f771914f4b837c9c85584c38ccf55a3ef3c2e8d154812246e5dda4a87450576b2c58ad9ab40c9e2edc31b288d066b195b21b',
-        filters => ['System.OS==linux'],
+        url      => 'http://example.com/asset/example.tar',
+        sha512   => '4f926bf4328fbad2b9cac873d117f771914f4b837c9c85584c38ccf55a3ef3c2e8d154812246e5dda4a87450576b2c58ad9ab40c9e2edc31b288d066b195b21b',
+        filters  => ['System.OS==linux'],
+        metadata => {
```
Comment (ghoneycutt) [inline spec/acceptance/sensu_asset_spec.rb]: Ugh, you have to specify this?

File: spec/acceptance/sensu_asset_spec.rb [hunk 2]
```diff
@@ -31,9 +36,12 @@
       pp = <<-EOS
       include ::sensu::backend
       sensu_asset { 'test':
-        url     => 'http://example.com/asset/example.tar',
-        sha512  => '4f926bf4328fbad2b9cac873d117f771914f4b837c9c85584c38ccf55a3ef3c2e8d154812246e5dda4a87450576b2c58ad9ab40c9e2edc31b288d066b195b21b',
-        filters => ['System.OS==windows'],
+        url      => 'http://example.com/asset/example.tar',
```
Comment (ghoneycutt) [inline spec/acceptance/sensu_asset_spec.rb]: should this be zip to match the content type?

File: spec/acceptance/sensu_asset_spec.rb (hunk 1 above)
Comment (treydock) [reply spec/acceptance/sensu_asset_spec.rb]: Don't have to, but I wanted to test that it works when specified.

File: spec/acceptance/sensu_asset_spec.rb (hunk 2 above)
Comment (treydock) [reply spec/acceptance/sensu_asset_spec.rb]: Technically, yes, but I wanted to test a modification to `metadata`.  I'll update to have file type match content type.

File: spec/acceptance/sensu_asset_spec.rb (hunk 1 above)
Comment (ghoneycutt) [reply spec/acceptance/sensu_asset_spec.rb]: Gotcha! Thought they required it which seemed crazy. I get it know.

File: spec/acceptance/sensu_asset_spec.rb (hunk 2 above)
Comment (ghoneycutt) [reply spec/acceptance/sensu_asset_spec.rb]: Please do and squash/rebase commits. Thanks!

**Comment by treydock** [comment]
PR updated, rebased and squashed.

**Pull request merged**